domain-model/src/test/java/uk/gov/hmcts/cmc/domain/models/InterestAmountTest.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.cmc.domain.models;
+
+import org.junit.Test;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class InterestAmountTest {
+
+    @Test
+    public void valueOfReturnObjectWithAmountEq0() {
```

Comment:
I guess we do not need this. It sort of suggests that there is special handling of zero values while only job that class is doing is you rounding half UP which might be documented in one tests case.